src/quadFeature.js

```diff
@@ -773,7 +786,9 @@ quadFeature.capabilities = {
   /* support for canvas elements as content in image quads */
   canvas: 'quad.canvas',
   /* support for parallelogram video quads */
-  video: 'quad.video'
+  video: 'quad.video',
+  /* support for raw texture data */
+  texture: 'quad.texture'
```

Comment:
We need to add `capabilities[quadFeature.capabilities.texture] = true;` near the end of src/webgl/quadFeature.js to indicate that this is supported (so programs that query by support get the webgl renderer).